ccd-definition/CaseEvent/CareSupervision/GatekeepingListing.json

```diff
@@ -6,7 +6,7 @@
     "Name": "Judicial Gatekeeping",
     "Description": "Judicial Gatekeeping",
     "DisplayOrder": 1,
-    "PreConditionState(s)": "Gatekeeping;PREPARE_FOR_HEARING",
+    "PreConditionState(s)": "Gatekeeping",
```

Comment:
Think in existing behaviour: if user upload an urgent direction then list a gate keeping hearing, the case will be progressed to "Case Management" state but still allow them to upload a gatekeeping order.

Reply:
ye part of the new requirement: "Once case is in case management state, 'List Initial Hearings' should not be displayed to any user in the next steps or hyperlink on the hearing Tab"